packages/patterns/src/patterns/patternMatchers.js

```diff
@@ -381,6 +382,9 @@ const makePatternKit = () => {
         // patterns
         return patt.every(checkIt);
       }
+      case 'byteArray': {
+        return true;
+      }
```

Comment:
ByteArrays are Keys, so this code should be unreachable as it is for e.g. "string".
```suggestion
```